billing_structs.go

```diff
@@ -55,6 +55,55 @@ func (m *PromoCodeClaim) Reset()         { *m = PromoCodeClaim{} }
 func (m *PromoCodeClaim) String() string { return proto.CompactTextString(m) }
 func (*PromoCodeClaim) ProtoMessage()    {}
 
+func (m *PromoCodeClaim) GetOrgId() int32 {
```

Comment:
These public functions probably belong in billing.go, and the structs in this file probably belong in pkg/structs/billing.go

Reply:
it's fine that public functions of a struct are put together with the definition of the struct.